_layouts/docwithnav.html

```diff
@@ -168,9 +191,40 @@
           </div>
           <div class="col-xs-12 col-sm-8 col-md-9">
             <div id="content-main">
+              <!-- TOP DOCUMENTATION PAGE NAVBAR -->
+              <div style="text-align: left; padding: 18px 0 9px 0; min-width: 500px; overflow-x: hidden;">
+              <ol class="breadcrumb">
+              <!-- START BREADCRUMBS -->
+				{% assign crumbs = page.url | remove:'/index.html' | split: '/' | shift | pop %}
+				{% if crumbs.first == site.data.versions.first[0] and crumbs[1] != 'usage' %}
+					{% assign crumbs_version = page.url | split: '/' | first %}
+					{% for crumb in crumbs offset: 1 %}
+					  {% capture crumb_url %}{{crumbs_version}}{% assign crumb_limit = forloop.index | plus: 1 %}{% for crumb in crumbs limit: crumb_limit %}{{ crumb | append: '/' }}{% endfor %}{% endcapture %}
+
+					  {% assign crumb_url_md = crumb_url | append: 'index.md' %}
+					  {% assign crumb_url_html = crumb_url | append: 'index.html' %}
+					  {% assign breadcrumb_exists = site.html_pages | where: "path", crumb_url_md %}
+					  {% if breadcrumb_exists == '' or breadcrumb_exists == nil or breadcrumb_exists.size == 0 %}
+					 	{% assign breadcrumb_exists = site.html_pages | where: "path", crumb_url_html %}
+					  {% endif %}
+
+					  {% if breadcrumb_exists != '' and breadcrumb_exists != nil and breadcrumb_exists.size > 0 %}
+						<li class="breadcrumb-item"><a href="{{site.baseurl}}/{{crumb_url}}">{{ crumb | replace:'-',' ' | remove:'.html' | capitalize }}</a></li>
+					  {% endif %}
+					{% endfor %}
+				 {% endif %}
+			  </ol>
+              <!-- END BREADCRUMBS -->
+              
+              <a href="https://github.com/projectcalico/calico/tree/master/{{page.path}}" style="color: #999; text-decoration: none; float: right;" target="_blank"><i class="fa fa-github" aria-hidden="true"></i> Edit</a></div>
```

Comment:
Is the leading space deliberate?  Should it be a&nbsp; as with the other heading links?